src/test/run-make/dump-mono-stats/foo.mono_items.md

```diff
@@ -0,0 +1,3 @@
+| Item | Instantiation count | Estimated Cost Per Instantiation | Total Estimated Cost |
```

Comment:
shouldn't that be the json file instead of the mardown one ?

Reply:
ah yeah my bad I didn't mean to commit this 😅